presto-cli/src/test/java/io/prestosql/cli/TestClientOptions.java

```diff
@@ -144,4 +154,12 @@ public void testEqualSignNoAllowedInPropertyCatalog()
     {
         new ClientSessionProperty(Optional.of("cat=alog"), "name", "value");
     }
+
+    @Test(expectedExceptions = IllegalArgumentException.class)
```

Comment:
Add `expectedExceptionsMessageRegExp` to verify the error message is the one we expect.